contracts/eosio.system/eosio.system-regproducer-rc.md

```diff
@@ -1,15 +1,63 @@
 # eosio.system regproducer
 
-This Ricardian contract for the system action *regproducer* is legally binding and can be used in the event of a dispute. 
+I, {{producer}}, hereby nominate myself for consideration as an elected block producer. 
 
-## regproducer
-    ( const account_name producer
-    , const public_key& producer_key
-    , const std::string& url
-    , uint16_t location );
+If {{producer}} is selected to produce blocks by the eosio contract, I will sign blocks with {{producer_key}} and I hereby certify that I will keep this key secret and secure. 
 
-_Intent: create a producer-config and producer-info object for 'producer'_
+If {{producer}} is unable to perform obligations under this contract I will resign my position by setting by resubmitting this contract with
```

Comment:
looks like half of this string should be removed: "by setting by resubmitting"

Reply:
yes